core/json/node.go

```diff
@@ -9,23 +9,24 @@ type PowDiff struct {
 
 // InfoNodeResult models the data returned by the node server getnodeinfo command.
 type InfoNodeResult struct {
-	ID               string              `json:"ID"`
-	Addresss         []string            `json:"address"`
-	QNR              string              `json:"QNR,omitempty"`
-	Version          int32               `json:"version"`
-	BuildVersion     string              `json:"buildversion"`
-	ProtocolVersion  int32               `json:"protocolversion"`
-	TotalSubsidy     uint64              `json:"totalsubsidy"`
-	GraphState       GetGraphStateResult `json:"graphstate"`
-	TimeOffset       int64               `json:"timeoffset"`
-	Connections      int32               `json:"connections"`
-	PowDiff          PowDiff             `json:"pow_diff"`
-	Network          string              `json:"network"`
-	Confirmations    int32               `json:"confirmations"`
-	CoinbaseMaturity int32               `json:"coinbasematurity"`
-	Errors           string              `json:"errors"`
-	Modules          []string            `json:"modules"`
-	DNS              string              `json:"dns,omitempty"`
+	ID               string                          `json:"ID"`
+	Addresss         []string                        `json:"address"`
+	QNR              string                          `json:"QNR,omitempty"`
+	Version          int32                           `json:"version"`
+	BuildVersion     string                          `json:"buildversion"`
+	ProtocolVersion  int32                           `json:"protocolversion"`
+	TotalSubsidy     uint64                          `json:"totalsubsidy"`
+	GraphState       GetGraphStateResult             `json:"graphstate"`
+	TimeOffset       int64                           `json:"timeoffset"`
+	Connections      int32                           `json:"connections"`
+	PowDiff          PowDiff                         `json:"pow_diff"`
+	Network          string                          `json:"network"`
+	Confirmations    int32                           `json:"confirmations"`
+	CoinbaseMaturity int32                           `json:"coinbasematurity"`
+	Errors           string                          `json:"errors"`
+	Modules          []string                        `json:"modules"`
+	DNS              string                          `json:"dns,omitempty"`
+	SoftForks        map[string]*SoftForkDescription `json:"softforks"`
```

Comment:
`SoftForkDescription` -> `ConsensusDeploymentDesc`  or `ConsensusDeployment` , is better?

Reply:
https://github.com/Qitmeer/qitmeer/pull/505/commits/9e050fb776b9f03e7561dcd244bfe995469db9fa